contracts/IController.sol

```diff
@@ -1,17 +1,21 @@
-pragma solidity ^0.5.11;
+// SPDX-FileCopyrightText: 2021 Livepeer <info@livepeer.org>
 
-import "./zeppelin/Pausable.sol";
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
We use the MIT license for Livepeer contracts right now so unless the code in a file is from a project that is using a different license, we should specify MIT for now.

Reply:
fixed in 1954d82